### DISCUSSION TITLE
juicefs log cannot rotate

### DISCUSSION THREAD

**Discussion by ZLEternity** [juicedata/juicefs#1990] [General]
/var/log/juicefs.log will grow infinitely when meta engine failure,  cannot rotate and clean

**Comment by davies** [comment]
It's recommended to rotate it using `logrotate`.

**Comment by qingyunha** [comment]
Do juicefs need a signal to reopen the log file?

**Reply by SandyXSD** [reply]
Nope. Please use `logrotate` with `copytruncate`.

**Reply by solracsf** [reply]
@SandyXSD what about include an example snippet of `logrotate` in docs?
